tests/test_matrix_utils.py

```
@@ -3131,3 +3224,107 @@ def import_side_effect(name):
             mock_logger.warning.assert_called_with(
                 "E2EE will be disabled for this session."
             )
+
+
+#!/usr/bin/env python3
```

Ruff (0.13.2)

Shebang is present but file is not executable

(EXE001)